browser/src/core/LOUtil.js

```diff
@@ -108,9 +108,15 @@ L.LOUtil = {
 		url += path;
 		return url;
 	},
-	setImage: function(img, name, doctype) {
-		img.src = this.getImageURL(name, doctype);
-		this.checkIfImageExists(img);
+	setImage: function(img, name, doctype, map) {
```

Comment:
`docType` can be extracted from `map.getDocType()`, maybe we can pass only map, for simpler usage?